config/samples/glance_v1beta1_glance.yaml

```diff
@@ -4,20 +4,17 @@ metadata:
   name: glance
 spec:
   serviceUser: glance
-  containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
   customServiceConfig: |
     [DEFAULT]
     debug = true
   databaseInstance: openstack
   databaseUser: glance
   glanceAPIInternal:
-    containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
     debug:
       service: false
     preserveJobs: false
     replicas: 1
   glanceAPIExternal:
-    containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
```

Comment:
@dprince I suspect also kuttl needs to be aligned with this change (as images are hardcoded)

Reply:
@fmount ack. Good call and thanks for pointing it out

Reply:
While kuttl is hard coded though, it should still align with the hard coded defaults in the Go library coded within this operator. So I think it should be okay right?

Reply:
Yeah, it makes sense: when we bump the `glanceAPI` image in the operator, we can update kuttl tests as well.
We should be good then from that point of view I guess.
Thanks for checking